programs/uxd/src/utils/math/compute_amount_after_change.rs

```diff
@@ -0,0 +1,28 @@
+use crate::error::UxdError;
+use anchor_lang::prelude::*;
+
+pub fn compute_amount_after_change(amount_before_change: u128, change: i128) -> Result<u128> {
```

Comment:
Makes things obfuscated I would argue, but we already discussed that and my involvement with this codebase is lower so ok for changes.
I would rename `compute_amount_after_change` to `compute_new_value(value: , delta:)` but nit and prob personal preference.

Reply:
yeah naming might not be ideal here indeed, seems like using an `unsigned_accumulator+adding_signed_delta` is a common pattern used a lot in the codebase, so it makes sense having nice utility function instead of inlining a lot of boilerplate imo, but yeah maybe we could stick with unsigned values across the codebase instead, then this function utility would become obsolete ?

Reply:
would the name "compute_amount_plus_delta" be more clear in this case you think ? "compute_new_value" feel a bit vague, and could refer to a lot of differen tthings

Reply:
maybe @GregoryNEUT has an opinion about this particular pattern? since you wrote that part of the codebase a while back

Reply:
hm, a lot opinionated. It add an extra layer level, but at same time reduce the boilerplate. Would say get rid of I80F48 is a good thing for now.

Let's go with it for now and see how the codebase evolve with new additions.

For naming idea: `safe_addition`